packages/components/src/components/list-box/_list-box.scss

```diff
@@ -86,6 +86,16 @@ $list-box-menu-width: rem(300px);
     }
   }
 
+  .#{$prefix}--list-box--xl {
+    height: rem(48px);
+    max-height: rem(48px);
```

Comment:
Seems like `max-height` is still being used? Context from previous review comments that were said to have been addressed: https://github.com/carbon-design-system/carbon/pull/4731/files#r349730150

Reply:
For this part, the list box gets shrunk down if we remove `max-height` rule here, given `.bx--list-box` has `max-height` rule.

Reply:
Got it, thanks!